fe/fe-core/src/main/java/org/apache/doris/load/ExportMgr.java

```diff
@@ -86,6 +96,44 @@ private void writeUnlock() {
         lock.writeLock().unlock();
     }
 
+    @Override
+    protected void runAfterCatalogReady() {
+        List<ExportJob> jobs = getExportJobs(JobState.PENDING);
+        // Because exportJob may be replayed from log
+        // we also need handle EXPORTING state exportJob.
+        jobs.addAll(getExportJobs(JobState.EXPORTING));
+        int runningJobNumLimit = Config.export_running_job_num_limit;
+        if (runningJobNumLimit > 0 && !jobs.isEmpty()) {
```

Comment:
No need to calc the remaining job num.
The number of thread in executor pool is limit to 5, and the submitted job will be queued when all threads are taken.
So here, we can simply do:
1. change the job's state to EXPORTING, so that it won't be scheduled again.
2. submit the job to the executor.

Reply:
Exporting state means this exportJob is exporting. If we change the job's state to `EXPORTING` here, the job may not be assigned to the thread in the executor pool. So I think the change of job's state should in ExportExportingTask.exec()